saio/container/Dockerfile

```diff
@@ -111,7 +111,7 @@ RUN pip install -r test-requirements.txt
 WORKDIR /home/swift
 RUN git clone https://github.com/NVIDIA/swift.git
 WORKDIR /home/swift/swift
-RUN git checkout $SwiftVersion
+RUN if [[ -n "$SwiftVersion" ]] ; then git checkout $SwiftVersion ; else git checkout $( git describe --tags --abbrev=0 $(git rev-list --tags --max-count=1) ) ; fi
```

Comment:
I assume this works...

Key thing, I think, that such a derivation would necessarily affect the Image "TAG" that gets specified as the result of the "docker build" command... perhaps that would necessitate a different approach than this (i.e. the version/tag derivation would be done in a "wrapper" around "docker build" that would, instead, explicitly specify the SwiftVersion arg to the Dockerfile... and, of course, specify the target name of the built Docker Image.